concepts/05 UI Components/TagBox/00 Getting Started with TagBox/60 Configure Multiple Tags Selection.md

```diff
@@ -0,0 +1,71 @@
+Use the following properties to adapt the TagBox text field to multiple selected tags:        
```

Comment:
What do you mean by "adapt"?

Reply:
Adapt here means 'to specify what to do when users select multiple tags and how the component should behave)